### ISSUE TITLE
[Feature Request] - Database Encryption

### ISSUE THREAD

**Issue by valentin-ivan** [sabrogden/Ditto#374]
Hi, 

I used Ditto for years at home and at work. Unfortunately about 1 year ago I was forced to stop using it at work because it does not ### encrypt his database file. Somehow I believed that it does, but actually it's only happening went sent over network. 
It will be a really nice feature, so please can you add this?

All the best!
Thanks!

**Comment by Lisakelly7** [comment]
Just download and use the Portable version and run it within a Truecrypt container.

**Comment by valentin-ivan** [comment]
Yeah. In the end I did something similar. Portable version inside Cryptomator Vault. 
Thanks!
